src/main/java/org/folio/circulation/domain/Loan.java

```diff
@@ -55,14 +60,18 @@
   private final ServicePoint checkoutServicePoint;
   private final ServicePoint checkinServicePoint;
 
-  private final LoanPolicy loanPolicy;
+  private final String overdueFinePolicyId;
+  private final String lostItemPolicyId;
+  private final Policies policies;
 
   private Loan(JsonObject representation, Item item, User user, User proxy,
-    ServicePoint checkinServicePoint, ServicePoint checkoutServicePoint,
-    DateTime originalDueDate, LoanPolicy loanPolicy,
-    Collection<Account> accounts) {
+               ServicePoint checkinServicePoint, ServicePoint checkoutServicePoint,
+               DateTime originalDueDate, Policies policies,
+               Collection<Account> accounts) {
 
-    requireNonNull(loanPolicy, "loanPolicy cannot be null");
+    this.policies = Optional.ofNullable(policies).orElse(new Policies());
+
+    requireNonNull(this.policies.getLoanPolicy(), "loanPolicy cannot be null");
```

Comment:
in case _policies_ is null _new Policies()_ will be created and loan policy will be null

Reply:
in this case loan policy = LoanPolicy.unknown with id = null

Reply:
Does it make sense to add check for LostItemPolicy and OverdueFinePolicy?

Reply:
Added